internal/controller/storageclient_controller.go

```diff
@@ -250,6 +275,15 @@ func (r *StorageClientReconciler) reconcileClusterResourceQuota(spec *quotav1.Cl
 	return nil
 }
 
+func (r *StorageClientReconciler) createOrUpdate(obj client.Object, f controllerutil.MutateFn) error {
+	result, err := controllerutil.CreateOrUpdate(r.ctx, r.Client, obj, f)
+	if err != nil {
+		return err
+	}
+	r.Log.Info("successfully created or updated", "operation", result, "name", obj.GetName())
```

Comment:
Can you please specialize this comment based on the result instead of having a generic message:
```suggestion
	r.Log.Info(fmt.Sprintf("%s successfully %s", obj.GetKind, result), "name", obj.GetName())
```

Reply:
done.